### PR TITLE
🐛 Fixed bug with zipping all files in the folder

### PR DESCRIPTION
**Pull request by shepilov**

### CODE REVIEW

File: tdrive/backend/node/src/services/documents/utils.ts [hunk 1]
```diff
@@ -351,25 +351,30 @@ export const addDriveItemToArchive = async (
...
+          child.id,
+          child,
+          archive,
+          repository,
+          context,
+          `${prefix || ""}${item.name}/`,
+        );
+      }
+      nextPage = items.nextPage?.page_token;
+    } while (nextPage > "");
```
Comment (ericlinagora) [inline tdrive/backend/node/src/services/documents/utils.ts]: `string > ""` ? is this for some special coercion rules like `null` or `false` or `"0"` do something interesting with `> ""`

Comment (shepilov) [reply tdrive/backend/node/src/services/documents/utils.ts]: `typeof nextPage!='undefined' && nextPage` doesn't look interesting, or for you do?

Comment (ericlinagora) [reply tdrive/backend/node/src/services/documents/utils.ts]: why not just `while (nextPage)` ? or `!!nextPage` if you want to make the coercion explicit ?

**Review by ericlinagora — APPROVED** [review]
(assuming the `file.name` -> `item.name` is correct)

**Pull request merged**